config/fieldparams/mainnet.go

```diff
@@ -33,7 +33,9 @@ const (
 	BlobSize                              = 131072        // defined to match blob.size in bazel ssz codegen
 	BlobSidecarSize                       = 131928        // defined to match blob sidecar size in bazel ssz codegen
 	KzgCommitmentInclusionProofDepth      = 17            // Merkle proof depth for blob_kzg_commitments list item
-	NextSyncCommitteeBranchDepth          = 5             // NextSyncCommitteeBranchDepth defines the depth of the next sync committee branch.
+	ExecutionBranchDepth                  = 4             // ExecutionBranchDepth defines the number of leaves in a merkle proof of the execution payload header.
+	SyncCommitteeBranchDepth              = 5             // SyncCommitteeBranchDepth defines the number of leaves in a merkle proof of a sync committee.
```

Comment:
The spec defines a `CurrentSyncCommitteeBranch` and `NextSyncCommitteeBranch`, but since the depth is the same, we might as well keep one variable. Even if we defined two separate types in the interfaces package, they would both alias `[5][32]byte` and thus would be assignable to each other, so compiler won't catch slip-ups.